src/ax/projection/consumer.go

```diff
@@ -21,9 +21,9 @@ type GlobalStoreConsumer struct {
 	stream messagestore.Stream
 }
 
-// Run reads pipes messages from the message stream to the projector until an
-// error occurs or ctx is canceled.
-func (c *GlobalStoreConsumer) Run(ctx context.Context) error {
+// Consume reads message from the store and forwards them to the projector until
```

Comment:
"reads **a** message from" or "reads message**s** from" ?

Reply:
Damn, I'm bad at this! :P